pom.xml

```diff
@@ -26,42 +36,372 @@
     <module>test</module>
     <module>jaxrs</module>
   </modules>
+  <distributionManagement>
+    <repository>
+      <id>jboss-releases-repository</id>
+      <name>JBoss Release Repository</name>
+      <url>https://repository.jboss.org/nexus/service/local/staging/deploy/maven2/</url>
+    </repository>
+    <snapshotRepository>
+      <id>jboss-snapshots-repository</id>
+      <name>JBoss Snapshot Repository</name>
+      <url>https://repository.jboss.org/nexus/content/repositories/snapshots/</url>
+    </snapshotRepository>
+  </distributionManagement>
 
   <properties>
-    <!-- integration tests are disabled by default, use profile to run 'arq' -->
+
+    <!-- END lra properties-->
+
+    <buildproperty.date>${maven.build.timestamp}</buildproperty.date>
+    <!-- Checkstyle is disabled by default, but it could be enabled for default by a particular module -->
+    <checkstyle.skip>true</checkstyle.skip>
+    <ipv6.server.jvm.args>-Djboss.bind.address=[::1] -Djboss.bind.address.management=[::1]
+      -Djboss.bind.address.unsecure=[::1] -Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=true</ipv6.server.jvm.args>
+    <jvm.args.byteman>-Dorg.jboss.byteman.verbose
+                        -Djboss.modules.system.pkgs=org.jboss.byteman
+                        -Dorg.jboss.byteman.transform.all
+                        -javaagent:${project.build.directory}/lib/byteman.jar=listener:true</jvm.args.byteman>
+    <jvm.args.debug></jvm.args.debug>
+
+    <!-- IP stack configs. -->
+    <jvm.args.ip>-Djava.net.preferIPv4Stack=true -Djava.net.preferIPv6Addresses=false</jvm.args.ip>
+    <jvm.args.jacoco>-javaagent:${project.build.directory}/lib/jacocoagent.jar=destfile=${project.build.directory}/coverage-reports/jacoco-ut.exec,includes=*,append=true,output=file</jvm.args.jacoco>
+    <!-- the need for -XX:MaxPermSize= was dropped as of Java SE 8 -->
+    <!-- -XX:ParallelGCThreads=2 is no not available in Java SE 9 -->
+    <jvm.args.memory>-Xms64m -Xmx1024m</jvm.args.memory>
+    <jvm.args.modular></jvm.args.modular>
+    <jvm.args.other>-server</jvm.args.other>
+
+    <!-- LRA properties-->
     <lra.test.timeout.factor>1.0</lra.test.timeout.factor>
 
+    <maven.build.timestamp.format>yyyy/MMM/dd HH:mm</maven.build.timestamp.format>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+
+    <narayana.version>7.1.0.Final</narayana.version>
```

Comment:
@marcosgopen at some point we may want to be able to build LRA with test builds of Narayana (like Narayana can build with test builds of the SPI) which will need changes in narayana.sh

Reply:
You are right, in that case we might need to override narayana.version
I'll try to add it here